pyingest/parsers/aps.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+
+import sys
+import json
+import codecs
+from adsputils import u2asc
+from jats import JATSParser
+
+class NoSchemaException(Exception):
+    pass
+
+class WrongSchemaException(Exception):
+    pass
+
+class UnparseableException(Exception):
+    pass
+
+
+class APSJATSParser(JATSParser):
+
+    def get_author_init(self,namestring):
+        output = u2asc(namestring)
+        for c in output:
+            if c.isalpha():
+                return c
+        return u'.'
+
+    def aps_journals(self, pid):
+#       mapping journal-meta/journal-id/publisher-id to bibstems
+        publisher_ids = {'PRL': 'PhRvL', 'PRX': 'PhRvX', 'RMP': 'RvMP',
```

Comment:
publisher_id would be better instantiated as a class variable (so it's not initialized in each function call) or even better, read from a config file.  This way addition of a new journal will not require a new code update